package.json

```diff
@@ -46,7 +46,6 @@
     "lint": "eslint .",
     "test": "mocha --require test/support/env --reporter spec --check-leaks --bail test/",
     "test-ci": "nyc --reporter=lcov --reporter=text npm test",
-    "test-cov": "nyc --reporter=html --reporter=text npm test",
-    "test-debug": "npm test -- --timeout 0"
```

Comment:
why did you remove this?

Reply:
I thought it was only added for debugging purposes. I can add it back if you want